backend/sam_gov_scraper.py

```
@@ -0,0 +1,573 @@
+
+def run_immediate_sam_cycle():
+    """Run immediate SAM.gov scraping cycle"""
+    db_path = os.path.join(os.path.dirname(__file__), 'instance', 'opportunities.db')
+    scraper = SamGovScraper(db_path)
+    
+    results = scraper.run_sam_scraping_cycle()
+    print(f"\n✅ Immediate SAM.gov cycle complete!")
+    print(f"🎯 Found {results['opportunities_found']} opportunities")
+    print(f"💾 Saved {results['opportunities_saved']} new opportunities")
+
+if __name__ == "__main__":
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)